pkg/cli/lambda_command.go

```
@@ -0,0 +1,391 @@
+		if err != nil {
+			return fmt.Errorf("failed to get connector manager client: %w", err)
+		}
+
+		// Get configuration, convert it to viper flag values, then proceed.
+		// TODO(morgabra): Should we start the lambda handler first? What are the timeouts for startup?
+		config, err := c1_lambda_config.GetConnectorConfig(ctx, client)
+		if err != nil {
+			return fmt.Errorf("failed to get connector config: %w", err)
+		}
+
+		fmt.Println("Received config: ", spew.Sdump(config))
```

golangci-lint (1.62.2)

use of `fmt.Println` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$`

(forbidigo)